projects/ltc7841/Makefile

```diff
@@ -0,0 +1,5 @@
+include ../../tools/scripts/generic_variables.mk
+
```

Comment:
Add:

```
EXAMPLE := iio_example
include ../../tools/scripts/examples.mk
```

These lines sets the default EXAMPLE in case not provided and includes the source/include files for the examples.